argo-cd-apps/base/host/toolchain-host-operator/toolchain-host-operator.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: toolchain-host-operator
+spec:
+  generators:
+    - merge:
+        mergeKeys:
+          - nameNormalized
+        generators:
+          - clusters:
+              values:
+                sourceRoot: components/sandbox/toolchain-host-operator
+                environment: staging
+                clusterDir: ""
+          - list:
+              elements: []
+  template:
+    metadata:
+      name: toolchain-host-operator-{{nameNormalized}}
+    spec:
+      project: default
+      source:
+        path: '{{values.sourceRoot}}/{{values.environment}}/{{values.clusterDir}}'
+        repoURL: https://github.com/redhat-appstudio/infra-deployments.git
+        targetRevision: main
+      destination:
+        namespace: toolchain-host-operator
+        server: '{{server}}'
+      syncPolicy:
+        automated:
+          prune: false
+          selfHeal: true
+        syncOptions:
+          - CreateNamespace=false
```

Comment:
@MatousJobanek could this be an issue - I mean the fact that the namespace for host and member operator are not managed by this application , they were created using our scripting ? I'm not sure if this is the cause of the e2e test failures.

Reply:
I don't thing that it could cause any issues. And yes, the toolchain operators are deployed via a logic that we have in toolchain-e2e repo, so the namespace should be present.

Reply:
Got it, thanks for answering!